data/mods/ssb/abilities.ts

```diff
@@ -231,51 +247,32 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 	youllneverwalkalone: {
 		shortDesc: "Boosts Atk, Def, SpD, and Spe by 25% under Anfield Atmosphere.",
 		name: "You'll Never Walk Alone",
```

Comment:
```suggestion
		name: "You'll Never Walk Alone",
		onStart(pokemon) {
			if (this.field.getPseudoWeather('anfieldatmosphere')) {
				this.add('-ability', pokemon, 'You\'ll Never Walk Alone');
			}
		},
```
maybe if we could get a way to detect when it ended to say the boosts were removed? idk thats a task for later